frontend/src/containers/dashboard/__snapshots__/index.spec.js.snap

```diff
@@ -565,95 +691,68 @@ exports[`Dashboard online logged in renders as expected 1`] = `
                   className="four wide column"
                 >
                   <DragInTrack
-                    disabled={false}
+                    disabled={true}
                     onDrop={[Function]}
                   >
-                    <Component
-                      backend={[Function]}
+                    <Header
+                      size="small"
                     >
-                      <DropTarget(UrlDropArea)
-                        accepts={
-                          Array [
-                            "__NATIVE_URL__",
-                          ]
-                        }
-                        onDrop={[Function]}
+                      <div
+                        className="ui small header"
                       >
-                        <UrlDropArea
-                          accepts={
-                            Array [
-                              "__NATIVE_URL__",
-                            ]
-                          }
-                          connectDropTarget={[Function]}
-                          onDrop={[Function]}
+                        Current Track
+                      </div>
+                    </Header>
+                    <Connect(CurrentTrackContainer)>
+                      <CurrentTrackContainer
+                        dispatch={[Function]}
+                        image={null}
+                        progress={NaN}
```

Comment:
I see this `progress` prop is `NaN` here in this snapshot.

Looking in `frontend/src/containers/current-track-container/index.js`, it looks like the `progress` prop value is provided by `timerToPercentage(state.timer)`, where `timerToPercentage()` is defined in `frontend/src/utils/time/index.js`:

```js
export const timerToPercentage = timer => {
  if (timer.duration === 0) { return 0 }
  const position = millisToSeconds(timer.position)
  const duration = millisToSeconds(timer.duration)

  return Math.round((position / duration) * 100)
}
```

Do we need to worry about `timerToPercentage()` returning `NaN` here? (`NaN` seems like a bad thing to return!)

Reply:
I suspect this is because the spec is not getting passed any values:
```
Math.round(undefined * 100)
NaN

Math.round((null / null) * 100)
NaN
```
I'll address this.